build-plugin/action.yml

```diff
@@ -81,6 +85,27 @@ runs:
         printf "plugin-id=%s\n" "${{ steps.package-plugin.outputs.plugin-id }}"
         printf "plugin-version=%s\n" "${{ steps.package-plugin.outputs.plugin-version }}"
 
+    - name: Checkout code
```

Comment:
at this point the code should have be already there (by the workflow invoking this action) it is part of our [example](https://github.com/grafana/plugin-actions/tree/main/build-plugin#workflow-example) on how to use this action and our templates in create-action